CONTRIBUTE.md

```diff
@@ -270,3 +270,11 @@ Here's a quick check list for a good pull request (PR):
 * New/changed features have been documented
 * A full build completes successfully
 * Do a rebase on upstream `master`
+
+### PR Handling (For Committers)
+
+* Don't merge your own PRs, ensure four eyes principle (unless it's trivial typo fixes and the like)
+* Ensure there are no "Octocat" commits
```

Comment:
Does everyone understand "Octocat commits"?

Reply:
Good point; made that a link to GH docs.